SRI-RSST/Phindr3D
# Segmentation unit testing

added unit testing to segmentation.py
added sample data for segmentation testing
added expected result data for comparison

test overview:
- load metadata for sample data
- segment sample data
- compare results to expected results
- remove the new image files created by running the segmentation

I tried to make the sample images as small as possible, but they're still pretty big for just testing purposes.
